pyproject.toml

```diff
@@ -16,7 +16,7 @@ classifiers = [
 ]
 requires-python = ">=3.10,<3.14"
 dependencies = [
-    "pymodbus (>=3.8.3,<3.10)",
+    "pymodbus>=3.10.0",
```

Comment:
Limit to <4. Who knows what breaking changes v4 might bring.

Reply:
should be changed